tests/test_language_id.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from floss.language_identifier import is_go_bin
+
+
+@pytest.mark.parametrize("binary_file, expected_result", [
+    ("./tests/data/src/go-hello/bin/go-hello.exe", True),
+    ("./tests/data/src/go-hello/bin/go-hello", False),
```

Comment:
should this be an expected fail for now, but actually `True`?

Reply:
Yes, it should be True. but elf not handled. maybe i will add it.

Reply:
let's comment it out with a note on that for now

Reply:
let's comment it out with a note on that for now